libs/vre/shared/app-resource-properties/src/lib/create-resource-form.component.ts

```diff
@@ -232,7 +232,11 @@ export class CreateResourceFormComponent implements OnInit {
     const propertiesObj: { [index: string]: CreateValue[] } = {};
 
     Object.keys(this.form.controls.properties.controls)
-      .filter(iri => this.form.controls.properties.controls[iri].controls.some(control => control.value.item !== null))
+      .filter(iri =>
+        this.form.controls.properties.controls[iri].controls.some(
+          control => control.value.item !== null && control.value.item !== ''
```

Comment:
this should stays null, because '' could be a valid value in some cases. I will propose a fix

Reply:
Also, I noticed that an exception is being raised when trying to save 0 as a decimal value:
@type: "http://www.w3.org/2001/XMLSchema#decimal"
@value: "0"

dsp.errors.InconsistentRepositoryDataException: Unexpected object of http://rdfh.ch/0811/LX1Y4tLSRiuxyKgw5ICpgQ/values/zRqfaZPGS0-qgtdQRNOYFA http://www.knora.org/ontology/knora-base#valueHasDecimal: 0",

Reply:
@derschnee68 We are merging this now, but I will leave the ticket open and assign it to you for later improvements.